### ISSUE TITLE
Compatability with ubuntu 18. 04

### ISSUE THREAD

**Issue by xianyitiantim** [lbr-stack/lbr_fri_ros2_stack#15]
Hi, may I know if I can use this package in ubuntu 18.04? I have melodic installed but I am really new at ros2. Thanks!

**Comment by anubhav-dogra** [comment]
you can use melodic version of this stack with "lbr_fri_ros_stack".
You can select melodic branch from all the branches provided.

**Comment by xianyitiantim** [comment]
@anubhav-dogra Thanks for your reply! I am so stupid I did not check! Sorry!

**Comment by xianyitiantim** [comment]
@anubhav-dogra Hi, when we are using this package to run Med 7, the robot would have some 'beep' noise. Have you encountered this problem before? Everything looks good except this issue.

**Comment by anubhav-dogra** [comment]
You might need to check errors or warnings on the KUKA SmartPad.

**Comment by mhubii** [comment]
I have noticed this before and it seems to happen occasionally. Not sure what the reason for this is. You might try different communication rates, which can be selected at start-up. Maybe try a lower one first. Control rates <2ms are only supported via the KONI port on the KUKA controller. Also, make sure you run a realtime linux kernel when you run high communication rates

**Comment by xianyitiantim** [comment]
@mhubii Typically I use 5 ms, I also tried 10 ms, and I do not think communication rates is an issue. I found this case always happens if I make the robot move fast. For example, after the robot moves, when I command it to return its original pose, I only give a desired position without controlling its velocity. Then the robot would make some noise. After I give a small velocity, this case rarely happens. Hope this can help!
